Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/TimeAndDateExtensions.kt

```diff
@@ -75,4 +76,12 @@ object TimeAndDateExtensions {
         val millionSeconds = secondDate - firstDate
         return TimeUnit.MILLISECONDS.toDays(millionSeconds)
     }
+
+    fun numberOfDayChanges(then: DateTime, now: DateTime = Instant().toDateTime(DateTimeZone.UTC)) =
+        Days.daysBetween(
+            then.withTime(0, 0, 0, 0),
+            now.withTime(0, 0, 0, 0)).days
+
+    fun numberOfDayChanges(millis: Long, now: DateTime = Instant().toDateTime(DateTimeZone.UTC)) =
+        numberOfDayChanges(Instant.ofEpochMilli(millis).toDateTime(DateTimeZone.UTC), now)
```

Comment:
Required: I find numberOfDayChanges confusing. To me, it's not clear whether changes is used as plural of change (I think it is, but I don't understand what the function is supposed to do then by reading the name) or as third-person singular of the verb (then I would expect it to return a boolean, but it doesn't).

What was wrong about ageInDays? If you find that confusing, then how about daysBetween (like the API it is wrapping) or daysBetweenIgnoreTime (inspired by Java's equalsIgnoreCase)?

Reply:
I like the later.